### PR TITLE
Rules-app

### PR DESCRIPTION
**Pull request by ayazsayyed**
Added sytles for slim scroll bar, margin for heading in modalpopup.

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -374,7 +374,15 @@ export const generate = (baseSpacing = 24, scale = 6) => {
     },
     box: {
       responsiveBreakpoint: 'small', // when we switch rows to columns
-      // extend: undefined,
+      extend: ({ theme }) => ({
+        '::-webkit-scrollbar': {
```
Comment (amitmishrg) [inline src/js/index.js]: Why have we added it for boxes?

File: src/js/index.js [hunk 2]
```diff
@@ -920,6 +928,7 @@ export const generate = (baseSpacing = 24, scale = 6) => {
           level: 3,
           size: 'small',
           color: 'dark-3',
+          margin: '0',
```
Comment (amitmishrg) [inline src/js/index.js:982]: lets discuss

File: src/js/index.js (hunk 1 above)
Comment (ayazsayyed) [reply src/js/index.js]: box is div element, adding slim scroll to divs with property of overflow

File: src/js/index.js (hunk 2 above)
Comment (ayazsayyed) [reply src/js/index.js:982]: can be ignored after discussion.

File: src/js/index.js (hunk 1 above)
Comment (ayazsayyed) [reply src/js/index.js]: implemented new "scrollablebox" key for scrollable divs

**Pull request merged**